nss-day-cohort-48/the-worst-site-ever-deguello
# Updated HTML and CSS

i updated both my html and css docs

##testing

1. check that links work
2. check that HTML and CSS are linked properly 

:)
